api/ocm/tools/signing/digest.go

```diff
@@ -51,6 +54,18 @@ func VerifyResourceDigest(cv ocm.ComponentVersionAccess, i int, bacc ocm.DataAcc
 		return false, nil
 	}
 
+	// Check if the resource has already been verified
+	store := general.Optional(ostore...)
+	if store != nil {
+		vcd := store.Get(cv)
+		if vcd != nil {
+			if vcd.Resources[i].Digest.Equal(raw.Digest) {
+				return true, nil
+			}
+			return false, fmt.Errorf("component version %s corrupted", common.VersionedElementKey(cv))
```

Comment:
isn't it kind of double response the same thing? `true + no error` vs. `false + error`
What does indicate, that the verification was ok? The boolean or the error? In other words: I would not return an error, when the digests are not equal as long as no error occurred (e.g. during digest calculation or store reading or some io-error ...)

Reply:
yes thats funny^^

> // The function returns true if the verification has been executed. If an error occurs, or
> // the verification has been failed, an appropriate error occurs.
> // If the resource is not signature relevant (false,nil) is returned.

I will adjust it to return `true` and adjust the error ^^